core/domain/config_domain_test.py

```diff
@@ -85,7 +85,7 @@ class ConfigPropertyRegistryTests(test_utils.GenericTestBase):
     """Tests for the config property registry."""
 
     def test_config_property_schemas_are_valid(self):
-        for property_name in config_domain.Registry._config_registry:  # pylint: disable=protected-access
```

Comment:
Don't do this. Implement a method in the Registry class that retrieves a list of all config property names, and use that in the test instead. Keep the data object private.

Reply:
Hey @seanlip, Do you mean adding something this to Registry class
``` python
@classmethod
    def get_config_registry(cls):
        return cls._config_registry
```
and calling this method in the tests?

Reply:
Yes, though you could probably just use `cls._config_registry.keys()`. Maybe call the method get_all_config_property_names, too.

Reply:
@seanlip Thanks for the help! but using  `cls._config_registry.keys()` caused a python lint error so, i used the previous approach. PTAL!

Reply:
Hm, I don't understand why it would cause a lint error. Could you explain what the lint error was, and why it triggered? I am pretty sure we use .keys() in several other places in the codebase without any issues.

Reply:
Hey @seanlip, I am getting this error PTAL!
```
Messages for Python 3 support:
************* Module oppia.core.domain.config_domain
W:300,15: dict.keys referenced when not iterating (dict-keys-not-iterating)

```

Reply:
How about list(cls._config_registry)?

Reply:
Done

Reply:
@seanlip PTAL